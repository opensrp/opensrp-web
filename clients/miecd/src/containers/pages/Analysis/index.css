.analysis {
  width: 100%;
  height: 85vh;
}

.analysis .analysis-iframe {
  position: relative;
  top: 20px;
  left: 0px;
  width: 100%;
  height: 80vh;
}

.analysis .analysis-title {
  margin-left: 24px;
}
