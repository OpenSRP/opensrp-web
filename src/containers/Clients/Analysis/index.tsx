import * as React from 'react';

import './index.css';

interface State {
  loading: boolean;
}

export class Analysis extends React.Component<{}, State> {
  constructor(props: any) {
    super(props);
    this.state = {
      loading: true,
    };
  }

  public render() {
    return (
      <div className="analysis-content">
        <div>
          <h2 id="analysis_title">Pregnancy - Analysis</h2>
        </div>
        <div className="analysis-wrapper">
          <iframe
            seamless={true}
            scrolling="yes"
            frameBorder="0"
            src="https://superset.ona.io/superset/dashboard/164/?standalone=true"
          />
        </div>
      </div>
    );
  }
}

export default Analysis;
