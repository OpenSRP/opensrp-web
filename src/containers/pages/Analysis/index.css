.analysis-content {
  width: 100%;
  height: 85vh;
}

.analysis-content iframe {
  position: relative;
  top: 20px;
  left: 0px;
  width: 100%;
  height: 80vh;
}

.analysis-content #analysis_title {
  padding: 0px 15px;
  border-left: 5px solid rgb(23, 181, 255);
}
