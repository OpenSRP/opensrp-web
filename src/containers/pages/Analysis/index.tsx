import React from 'react';
import './index.css';

export default function Analysis() {
  return <div>this is the Analysis page</div>;
}
