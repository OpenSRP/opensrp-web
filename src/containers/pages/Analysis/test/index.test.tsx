import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import Analysis from '..';

describe('Analysis', () => {
  it('must render without crashing', () => {
    shallow(<Analysis />);
  });

  it('must render correctly', () => {
    const wrapper = mount(<Analysis />);
    expect(toJson(wrapper)).toMatchSnapshot('Analysis snapshot');
  });
});
