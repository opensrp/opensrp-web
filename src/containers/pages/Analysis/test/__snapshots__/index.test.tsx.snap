// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Analysis must render correctly: Analysis snapshot 1`] = `
<Analysis>
  <div
    className="analysis-content"
  >
    <div>
      <h2
        id="analysis_title"
      >
        Pregnancy - Analysis
      </h2>
    </div>
    <div
      className="analysis-wrapper"
    >
      <Ripple
        borderColor="#ff5d00"
        borderStyle="solid"
        borderWidth="4px"
        height="64px"
        minHeight="80vh"
        width="64px"
      >
        <div
          className="lds-ripple-parent"
          style={
            Object {
              "minHeight": "80vh",
            }
          }
        >
          <div
            className="lds-ripple"
            style={
              Object {
                "height": "64px",
                "width": "64px",
              }
            }
          >
            <div
              style={
                Object {
                  "borderColor": "#ff5d00",
                  "borderStyle": "solid",
                  "borderWidth": "4px",
                }
              }
            />
            <div
              style={
                Object {
                  "borderColor": "#ff5d00",
                  "borderStyle": "solid",
                  "borderWidth": "4px",
                }
              }
            />
          </div>
        </div>
      </Ripple>
      <iframe
        frameBorder="0"
        onLoad={[Function]}
        scrolling="yes"
        seamless={true}
        src="https://discover.ona.io/superset/dashboard/53/?standalone=true"
        title="pregnancy sms events analysis"
      />
    </div>
  </div>
</Analysis>
`;
